msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/BrokerTestHelper.java

```diff
@@ -22,41 +22,51 @@
 //  THE SOFTWARE.
 package com.microsoft.identity.client.msal.automationapp;
 
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
 import com.microsoft.identity.client.msal.automationapp.BuildConfig;
 import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.broker.AbstractTestBroker;
 import com.microsoft.identity.client.ui.automation.broker.BrokerCompanyPortal;
 import com.microsoft.identity.client.ui.automation.broker.BrokerHost;
 import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator;
 import com.microsoft.identity.client.ui.automation.broker.ITestBroker;
+import com.microsoft.identity.client.ui.automation.installer.IAppInstaller;
 
 import java.util.Arrays;
 import java.util.List;
 
 public class BrokerTestHelper {
-    public static ITestBroker createBrokerFromFlavor(@Nullable final SupportedBrokers supportedBrokersAnnotation) {
+    public static ITestBroker createBrokerFromFlavor(@Nullable final SupportedBrokers supportedBrokersAnnotation, @NonNull final IAppInstaller appSource) {
+        return internalCreateBrokerFromFlavor(supportedBrokersAnnotation, appSource);
+    }
+    public static ITestBroker createBrokerFromFlavor(@Nullable final SupportedBrokers supportedBrokersAnnotation){
```

Comment:
new line between each method

```suggestion
    }
    
    public static ITestBroker createBrokerFromFlavor(@Nullable final SupportedBrokers supportedBrokersAnnotation){
```

Reply:
30bc409603c6343679d468a20f055802036cac8d